lib/cfg-tree.c

```diff
@@ -1065,6 +1066,38 @@ cfg_tree_compile(CfgTree *self)
   return TRUE;
 }
 
+static gboolean
+verify_unique_persist_ids_among_pipes(const GPtrArray *initialized_pipes)
```

Comment:
and please prefix the function with an underscore to indicate that it is a private.